### DISCUSSION TITLE
DOTMIM

### DISCUSSION THREAD

**Discussion by MHefny123** [Mimetis/Dotmim.Sync#687] [Q&A]
is there a way to set a sql db as server hub and sync uploads & downloads between the hub database and more than one client databases three or more

**Comment by Mimetis** [comment]
Of course :)
- 1 server
- N clients

**Reply by MHefny123** [reply]
I have a setup like below Server db(Hub)
client 1 , client 2 , client x 
when syncing client 1 upload and download going correctly 
then syncing client 2 upload going correctly but can not download the client one  uploads  to be down on client two
here is my Issue
